docs/about-lightning.md

```diff
@@ -0,0 +1,228 @@
+---
+title: Lightning (Coming soon)
+sidebar_label: Lightning
+---
+
+## The Lightning pitch
+
+##### Leveraging the tech powering the field-tested enterprise OpenFn platform...
+
+Lightning brings together the tried and tested technology we have been using
+since 2015 (OpenFn core, engine, adaptors and code from the OpenFn SaaS
```

Comment:
Can we just say `the OpenFn Integration Toolkit Digital Public Good` instead of listing the components only our team knows about? `(OpenFn core, engine, adaptors and code from the OpenFn SaaS platform)`